src/interfaces/hapi.ts

```diff
@@ -32,9 +36,9 @@ var packageJson = require('../../../package.json');
  * @returns {ErrorMessage} - a partially or fully populated instance of
  *  ErrorMessage
  */
-function hapiErrorHandler(req, err, config) {
+function hapiErrorHandler(req: hapi.Request, err: {}, config: Configuration) {
```

Comment:
Can we use `Error` for `err`?

Reply:
I think this is the same as with Express.